src/hooks/subgraph/useSubgraphBets.ts

```diff
@@ -14,15 +14,17 @@ const POLL_INTERVAL = 5000;
 
 export const useSubgraphBets = (
   myBetsEnabled: boolean,
-  filter: BetFilterOptions
+  filter: BetFilterOptions,
+  marketId?: string
 ) => {
   const { address } = useAccount();
   const [betHistory, setBetHistory] = useState<BetHistory[]>();
 
   const { data, refetch } = useSubgraph<Response>(
     utils.queries.getBetsQuery({
       address: myBetsEnabled ? address : undefined,
-      filter
+      filter,
+      marketId: marketId ? marketId : undefined
```

Comment:
i think this can just be `marketId` as the type for the argument is `string | undefined`:
```ts
address: myBetsEnabled ? address : undefined,
filter,
marketId
```